composer.json

```diff
@@ -11,7 +11,9 @@
         "roave/signature": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.3.0"
+        "phpunit/phpunit": "^6.3.0",
+        "slevomat/coding-standard": "^3.2.1",
+        "squizlabs/php_codesniffer": "^3.0.1"
```

Comment:
I suggest explicitly using `composer require slevomat/coding-standard:^3.2.1 squizlabs/php_codesniffer:^3.0.1` as a CLI command in the build pipeline step.

Reply:
It's really difficult to check CS before push then :(

Reply:
Yeah, it's annoying, but since these are also reflection-ish tools, the dependencies can conflict at some point, making `--dev` installation differ too much from `--no-dev`. I think this is acceptable, as CS can be easily ffixed anyway.

Reply:
Done.